src/mbgl/renderer/buckets/line_bucket.cpp

```diff
@@ -392,14 +427,18 @@ void LineBucket::addGeometry(const GeometryCoordinates& coordinates, const Geome
 }
 
 void LineBucket::addCurrentVertex(const GeometryCoordinate& currentCoordinate,
-                                  double &distance,
+                                  double distance,
```

Comment:
Why do we no longer need the distance as a reference? We still seem to be modifying it.

Reply:
Thanks for the catch! I missed that we need to set it to `0.0` under some conditions